ambry-server/src/integration-test/java/com.github.ambry.server/ServerHardDeleteTest.java

```diff
@@ -323,7 +326,7 @@ public void endToEndTestHardDeletes() throws Exception {
     notificationSystem.awaitBlobDeletions(blobIdList.get(6).getID());
 
     time.sleep(TimeUnit.DAYS.toMillis(1));
-    ensureCleanupTokenCatchesUp(chosenPartition.getReplicaIds().get(0).getReplicaPath(), mockClusterMap, 298081);
+    ensureCleanupTokenCatchesUp(chosenPartition.getReplicaIds().get(0).getReplicaPath(), mockClusterMap, 298732);
```

Comment:
Same here - would be good if we can account clearly for the increase of 651 bytes here.

Reply:
same comment as above

Reply:
Similar math here.
1 Delete
3 puts (2 encrypted)
2 Delete

6*4 (header) + 3*1 ( blob props) + 114*2 = 225 + 376 (from previous checkpoint) = 631